src/main/java/net/minecraftforge/common/extensions/IForgeBlock.java

```diff
@@ -729,14 +689,21 @@ default boolean shouldDisplayFluidOverlay(BlockState state, BlockAndTintGetter w
      * @param pos The block position in world
      * @param player The player clicking the block
      * @param stack The stack being used by the player
+     * @param toolAction
      * @return The resulting state after the action has been performed
      */
     @Nullable
-    default BlockState getToolModifiedState(BlockState state, Level world, BlockPos pos, Player player, ItemStack stack, ToolType toolType)
-    {
-        if (toolType == ToolType.AXE) return AxeItem.getAxeStrippingState(state);
-//        else if(toolType == ToolType.HOE) return HoeItem.getHoeTillingState(state); //TODO HoeItem bork
-        else return toolType == ToolType.SHOVEL ? ShovelItem.getShovelPathingState(state) : null;
+    default BlockState getToolModifiedState(BlockState state, Level world, BlockPos pos, Player player, ItemStack stack, ToolAction toolAction)
+    {
+        if (!stack.canPerformAction(toolAction)) return null;
+        if (ToolActions.STRIP.equals(toolAction)) return AxeItem.getAxeStrippingState(state);
+        else if(ToolActions.SCRAPE.equals(toolAction)) return WeatheringCopper.getPrevious(state).orElse(null);
+        else if(ToolActions.WAX_OFF.equals(toolAction)) return Optional.ofNullable(HoneycombItem.WAX_OFF_BY_BLOCK.get().get(state.getBlock())).map((p_150694_) -> {
+            return p_150694_.withPropertiesOf(state);
+        }).orElse(null);
+        //else if(ToolActions.STRIP.equals(toolAction)) return HoeItem.getHoeTillingState(state); //TODO HoeItem bork
```

Comment:
I know its commented out, but that says STRIP instead of TILL

Reply:
Will be fixed on the next commit -- whenever that happens.